docker/docker-compose.etna-app.shared.yml

```diff
@@ -0,0 +1,90 @@
+
```

Comment:
One of the nice benefits of the changes I've made here include being able to extract the common docker-compose.yml logic that had previously been duplicated many places into a common shared file.

Technically, this still gets expanded explicitly into each project's yml file (see default_compose script) through a merge process, we are now able to add or change defaults here and have them propogate equally across other yml files easily.  yml by default does not include `include` or `require` type mechanics, so this is the closest we can get to that.